readme.md

```diff
@@ -499,26 +499,9 @@ CoIdentity(1).extend((co) => co.extract() + 1) // CoIdentity(2)
 
 ## Applicative Functor
 
-An applicative is a monoidal [functor](#functor). That is, applicative structures represent [monoids](#monoid) for which function application itself is an identity. This sounds complicated, but it really just requires abstract thinking and a few specific examples:
+An applicative functor is technically a [functor](#functor) that also has the properties of a [monoid](#monoid). More practically, it is a structure that allows for the application of functions within a context (for example, an array or list) to values within the same type of context. By implementing an applicative functor, you are in essence overloading the function operator `()` itself. Such an implementation will typically define an `ap` method on applicative functor objects that specifies what function application _means_ for the context that object represents. To offer a simple analogy, the plus operator `+` defines what addition _means_ for numbers. Similarly, the `concat()` function defines what addition _means_ for arrays. With an applicative functor, you can essentially generalize what addition means for any kind of structure or value. If you think of numbers as sharing the context of "numeric arithmetic" with addition and arrays as sharing the context of "array arithmetic" with `concat()`, then an applicative functor expresses how it shares the context of "arithmetic for this kind of thing" with function application in general, as defined by the `ap` method.
```

Comment:
Giant wall of text. Also still confusing. 

I'm still not convinced that describing the monoid relationship of applicative is illuminating for newcomers. 

We could also describe monad as a monoid in the catagory of endofunctors but getting that abstract scares people away fast.